README.md

```diff
@@ -45,7 +87,9 @@ git clone https://github.com/Cloud-CV/evalai-cli.git EvalAI-CLI
 ```bash
 cd EvalAI-CLI
 virtualenv -p python3 venv
-source venv/bin/activate
+cd venv/scripts
```

Comment:
@gautamjajoo I think on Windows the 'S' in `scripts` is capital. 
Also, how about the one-liner `venv\Scripts\activate`?
This is the recommended syntax. ([reference](https://virtualenv.pypa.io/en/stable/userguide/#activate-script))